setup.py

```diff
@@ -10,7 +10,7 @@
 
 from setuptools import find_packages, setup
 
-current_date = datetime.now().strftime("%Y.%m.%d")
+current_date = datetime.now().strftime("%Y%m%d")
```

Comment:
heads up to @joecummings and @ebsmothers in case you have version guards on nightly date

Reply:
Yep, we'll catch and fix